package.json

```diff
@@ -95,6 +95,7 @@
     "openlayers": "3.15.1",
     "pdfviewer": "0.3.2",
     "proj4": "2.3.14",
+    "qrcode.react": "^0.6.1",
```

Comment:
Domani fixo
Il 05/lug/2016 18:24, "Lorenzo Natali" <notifications@github.com> ha
scritto:

> In package.json
> <https://github.com/geosolutions-it/MapStore2/pull/808#discussion_r69594055>
> :
>
> > @@ -95,6 +95,7 @@
> >      "openlayers": "3.15.1",
> >      "pdfviewer": "0.3.2",
> >      "proj4": "2.3.14",
> > +    "qrcode.react": "^0.6.1",
>
> This should be fixed to 0.6.1
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/geosolutions-it/MapStore2/pull/808/files/261b03642855cc4743c487f126be075a18dee52c#r69594055>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ALb5hDZObgkCJhmJ9d61RtZIxI_D5PqBks5qSoVGgaJpZM4JFM1j>
> .
>